packages/lib/models/settings/builtInMetadata.ts

```diff
@@ -498,6 +498,36 @@ const builtInMetadata = (Setting: typeof SettingType) => {
 			isGlobal: true,
 		},
 
+		'ocr.languageDataPath': {
+			value: '',
+			type: SettingItemType.String,
+			advanced: true,
+			public: true,
+			appTypes: [AppType.Desktop],
+			label: () => _('OCR: Language data URL'),
+			description: () => _('A URL or path to a directory with language data files for OCR. After the first download, language data files will be cached to the file system. Leave this blank to download OCR language data from the default website.'),
```

Comment:
Could you move this explanation to the documentation please? Since it's quite long to translate and won't be used much probably